source/viz.md

```diff
@@ -1896,6 +1896,11 @@ import numpy as np
 png_size = np.round(os.path.getsize("img/faithful_plot.png")/(1024*1024), 2)
 svg_size = np.round(os.path.getsize("img/faithful_plot.svg")/(1024*1024), 2)
 
+png_size, svg_size
+```
+
+```{code-cell} ipython3
+:tags: [remove-cell]
```

Comment:
These are the only important changes -- the rest are whitespace (sorry, will get a pr for that soon!)